sdk/core/azure-core/src/main/java/com/azure/core/util/polling/Poller.java

```diff
@@ -361,6 +361,7 @@ private boolean matchStatus(PollResponse<T> currentPollResponse, OperationStatus
             .onErrorResume(throwable -> {
                 // We should never get here and since we want to continue polling
                 //Log the error
```

Comment:
You can remove this line //Log the error

Reply:
Done.